src/components/Toolbar.js

```diff
@@ -195,6 +195,12 @@ export default class Toolbar extends Component {
     return null;
   }
 
+  hasStyle(style) {
+    const current = this.props.editorState.getCurrentInlineStyle();
+    const withStyle = current.filter((val, key) => key.startsWith(style));
+    return withStyle.size > 0;
+  }
+
```

Comment:
Does active state have a test?  If so, this probably deserves a test for the bugfix 🙌

Reply:
nope it does not currently have a test. :)